packages/react-self-serializers/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@shopify/react-self-serializers",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "A set of self-serializing providers for @shopify/react-html applications",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-self-serializers/README.md",
+  "dependencies": {
+    "@shopify/react-html": "8.1.3",
```

Comment:
Such specific versions almost guarantees duplicate `@shopify/*` libs for consumers.  Can these be looser?

Reply:
Isn't part of the idea of lerna that it updates these deps for us? I'm not sure what affect loosening these would have on how it acts tbh

Reply:
>  I'm not sure what affect loosening these would have on how it acts tbh

Mandating specific versions will force consumers to install _exactly_ that version.  They might know how to dedupe dependencies, but historically most people don't even notice duplication.

Here are examples of quilt projects that use loose requirements for internal dependencies:

https://github.com/Shopify/quilt/blob/caabc13fd71cd2acde3e7a78e722e58f187bdce3/packages/react-form/package.json#L38-L43

https://github.com/Shopify/quilt/blob/caabc13fd71cd2acde3e7a78e722e58f187bdce3/packages/react-html/package.json#L25-L31

https://github.com/Shopify/quilt/blob/caabc13fd71cd2acde3e7a78e722e58f187bdce3/packages/react-i18n/package.json#L36-L42

https://github.com/Shopify/quilt/blob/caabc13fd71cd2acde3e7a78e722e58f187bdce3/packages/react-idle/package.json#L26-L28

https://github.com/Shopify/quilt/blob/caabc13fd71cd2acde3e7a78e722e58f187bdce3/packages/react-async/package.json#L26-L34

Reply:
sorry I meant how it effects lerna's auto-updating of the versions

Reply:
Adding the caret seems fine though regardless